src/main/scala/codacy/plugins/test/JsonTests.scala

```diff
@@ -12,27 +12,28 @@ object JsonTests extends ITest {
 
   val opt = "json"
 
-  def run(testSources: Seq[Path], dockerImage: DockerImage, optArgs: Seq[String]): Boolean = {
-    Printer.green("Running JsonTests:")
+  def run(testDirectories: Seq[Path], dockerImage: DockerImage, optArgs: Seq[String]): Boolean = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method is longer than 50 lines](https://app.codacy.com/app/Codacy/codacy-plugins-test/pullRequest?prid=4500290)